packages/sources/svelte/src/components/indicators/VtmnProgressbar/VtmnProgressbarLinear.svelte

```diff
@@ -45,17 +45,20 @@
     indeterminate && `vtmn-progressbar--indeterminate`,
     className,
   );
+
+  $: ariaProps = {
+    'aria-labelledby': labelId ? labelId : undefined,
```

Comment:
@lauthieb I don't know if a rework is needed here, but finally, we can use directly the `aria-labelledby` from the `$$restProps` ? 

Or do you think it's better to keep like that ? 🤔 

Otherwise, we can set the `export let labelId = undefined` and remove the ternary line 50 :)

Reply:
Hi @Tlahey,
the idea behind this rework was to harmonize code with Vue and React. 
I agree with you to explicitly default the labelId to `undefined` directly in the props though. A default value is set on the parent `VtmnProgressbar` element, but not explicit in this `VtmnProgressbarLinear`.